gef.py

```diff
@@ -11369,6 +11327,32 @@ def chunks(self) -> Union[List, Iterator]:
             return []
         return iter(GlibcChunk(self.base_address, from_base=True))
 
+    @property
+    def min_chunk_size(self) -> int:
+        return 4 * gef.arch.ptrsize
+
+    @property
+    def malloc_alignment(self) -> int:
+        __default_malloc_alignment = 0x10
+        if get_libc_version() > (2, 25) and is_x86_32():
+            # Special case introduced in Glibc 2.26:
+            # https://elixir.bootlin.com/glibc/glibc-2.26/source/sysdeps/i386/malloc-alignment.h#L22
+            return __default_malloc_alignment
+        # Generic case:
+        # https://elixir.bootlin.com/glibc/glibc-2.26/source/sysdeps/generic/malloc-alignment.h#L22
+        return 2 * gef.arch.ptrsize
+
+    def csize2tidx(self, size: int) -> int:
+        return abs((size - self.min_chunk_size + self.malloc_alignment - 1)) // self.malloc_alignment
+
+    def tidx2size(self, idx: int) -> int:
+        return idx * self.malloc_alignment + self.min_chunk_size
+
+    def malloc_align_address(self, address: int) -> int:
```

Comment:
funny, I had the same changes in my upcoming PR. but I noticed that `malloc_align_address` is superfluous. instead we can use the already existing function for this:

```py
align_address_by_size($whatever, gef.heap.malloc_alignment)
```

I would instead slightly adjust the comment and move it to `gef.heap.malloc_alignment` for reference.

Reply:
Those heap functions were becoming necessary and cheap to add 😄 
If you review `tests/commands/heap.py`, bear in mind it's not 100% complete for now it's just a better port of what was in the old `runtests.py`